ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/HttpRequestImpl.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2020 Confluent Inc.
```

Comment:
nit, same for HttpResponseImpl:

```suggestion
 * Copyright 2021 Confluent Inc.
```